# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/CompileWrappers.cpp
# Commit: 8129ee5398d5
# Full Hash: 8129ee5398d5ce34183dd2ca51c3aa1d27e9c318
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2023-01-20 03:46:23
# Regressor Bug: 1809359
# File Overlap Count: 1
# Description:
#   Bug 1809359 - Inline SetElementMegamorphic r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166401
# ==============================================================================

diff -r a4156a9d0b2b -r 8129ee5398d5 js/src/jit/CompileWrappers.cpp
--- a/js/src/jit/CompileWrappers.cpp	Thu Jan 19 22:22:32 2023 +0000
+++ b/js/src/jit/CompileWrappers.cpp	Thu Jan 19 23:53:40 2023 +0000
@@ -80,6 +80,10 @@
   return &runtime()->caches().megamorphicCache;
 }
 
+const void* CompileRuntime::addressOfMegamorphicSetPropCache() {
+  return &runtime()->caches().megamorphicSetPropCache;
+}
+
 const void* CompileRuntime::addressOfStringToAtomCache() {
   return &runtime()->caches().stringToAtomCache;
 }